hostthedocs/filekeeper.py

```diff
@@ -15,15 +15,46 @@ def sort_by_version(x):
     return x['version'].replace('.', '~') + 'z'
 
 
+def is_valid_doc_folder(folder):
```

Comment:
I have renamed the function from `_is_valid_doc_folder` to `_is_valid_doc_version` since the function is not tests the project root folder, but the folder with the version name.